src/molecules/header.tsx

```diff
@@ -28,14 +29,20 @@ const HeaderWrapper = styled.div`
   display: flex;
   align-items: center;
   flex-direction: column;
-  padding-top: 3%;
+  padding-top: 2%;
   @media (max-width: ${({ theme }) => theme.breakpoints.sm + 'px'}) {
-    width: 90%;
+    width: 80%;
+    padding-top: 0%;
+    padding-bottom: 10%;
   }
 `;
 
 const Logo = styled(Img)`
-  width: 100%;
+  width: 90%;
+  @media (max-width: ${({ theme }) => theme.breakpoints.sm + 'px'}) {
+    width: 90%;
+    display: none;
```

Comment:
Maybe the filterbar should stick to the top on scrolling instead? It would be nice to see the logo at first

Reply:
The filter bar cannot be in the header if only the filterbar should stick to the top when scrolling. Should I remove it from the header?